### PR TITLE
[Pool Simulator] Add points as pool limit in both curves

### PR DESCRIPTION
**Pull request by yvesfracari**
https://www.loom.com/share/bc97b64979e44387a9f43182972ef656?sid=aa387335-9aa0-451a-98d8-8982d18ef5d0

Also, rename `variant` to `custom` identification on Price Impact Curve

### CODE REVIEW

**Review by ribeirojose — APPROVED** [review]

File: apps/balancer-tools/src/app/poolsimulator/(components)/ImpactCurve.tsx [hunk 1]
```diff
@@ -230,7 +291,7 @@ const calculateTokenImpact = ({
 }: {
   tokenIn: TokensData;
   tokenOut: TokensData;
-  amm: AMM<MetaStablePoolPairData>;
+  amm: AMM<PoolPairData>;
```
Comment (luizakp) [inline apps/balancer-tools/src/app/poolsimulator/(components)/ImpactCurve.tsx:294]: 👍

**Review by luizakp — APPROVED** [review]
Great! The scale needs to be readjusted on the swap curve to when Gyro Pools are the smallest. It's something that I would work on this PR, but not exactly a blocker too

https://www.loom.com/share/bfcaefa93bd049838b88844a8e25c14e?sid=2d574c14-1d0e-4a86-acee-8bba2fd1c01f

**Comment by luizakp** [comment]
> Great! The scale needs to be readjusted on the swap curve to when Gyro Pools are the smallest. It's something that I would work on this PR, but not exactly a blocker too
> 
> https://www.loom.com/share/bfcaefa93bd049838b88844a8e25c14e?sid=2d574c14-1d0e-4a86-acee-8bba2fd1c01f

@yvesfracari solved this c:

**Comment by ribeirojose** [comment]
great work here folks 😄

**Pull request merged**